share/chruby/chruby.sh

```diff
@@ -42,13 +58,12 @@ function chruby_use()
 	export RUBY_ROOT="$1"
 	export RUBYOPT="$2"
 	export PATH="$RUBY_ROOT/bin:$PATH"
+	local ruby_cached_env="$RUBY_ROOT/.ruby_env"
```

Comment:
You won't always have write access to the ruby's root.

Reply:
If this were made configurable by the user (opting out of it e.g. by not providing a `RUBY_ENV_CACHE`), could it be merged then?